controllers/humiocluster_controller_test.go

```diff
@@ -106,6 +107,15 @@ var _ = Describe("HumioCluster Controller", func() {
 					}
 
 					_ = k8sClient.Delete(context.Background(), &cluster)
+
+					if cluster.Spec.License.SecretKeyRef != nil {
+						k8sClient.Delete(context.Background(), &corev1.Secret{
```

Comment:
I'm considering if we should introduce a linting rule such that we enforce return values to be assigned to a variable. The main goal for that is to make sure we don't accidentally skip handling an error if we should have.

That would mean this line instead would require this change, since we don't really care about the output of deleting the object in this specific line:
```suggestion
						_ = k8sClient.Delete(context.Background(), &corev1.Secret{
```
I'm not sure about the details about how to actually enforce it, but what do you think about introducing such a rule?

Reply:
Linting sounds good to me. Perhaps another CI job?